### PR TITLE
Implement Custom Log File #2

### PR DESCRIPTION
**Pull request by LKDevelopment**
Hello,

Happy #Hacktoberfest so let's start with some issues :)

This little PR implements the possibility to use custom log files. You can add a path to a custom file and than it will log to the file there :)

### CODE REVIEW

File: src/DeleteOldSoftDeletes.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 use Illuminate\Database\Eloquent\SoftDeletes;
 use Illuminate\Support\Collection;
 use Illuminate\Support\Facades\Log;
+use Monolog\Logger;
```
Comment (mattstauffer) [inline src/DeleteOldSoftDeletes.php]: Can drop this import.

File: src/DeleteOldSoftDeletes.php [hunk 2]
```diff
@@ -75,6 +76,10 @@ private function logAffectedRows(Collection $deletedRows)
         if (! $this->config->get('quicksand.log', false) || empty($preparedRows)) {
             return;
         }
+        
+        if (! $this->config->get('quicksand.custom_log_file', false)) {
```
Comment (mattstauffer) [inline src/DeleteOldSoftDeletes.php]: I think you mean this to be `!!` ?

File: src/config/quicksand.php [hunk 3]
```diff
@@ -7,6 +7,10 @@
     // Whether to log the number of soft deleted records per model
     'log' => false,
 
+    // If you log the soft deleted records per model, this is the path where it will be stored
+    // false if you want the default laravel log file
```
Comment (mattstauffer) [inline src/config/quicksand.php]: Maybe add a third line of comment showing an example path

File: src/DeleteOldSoftDeletes.php (hunk 1 above)
Comment (LKDevelopment) [reply src/DeleteOldSoftDeletes.php]: Sure, will drop this.

File: src/DeleteOldSoftDeletes.php (hunk 2 above)
Comment (LKDevelopment) [reply src/DeleteOldSoftDeletes.php]: Oh Damn... yeahr sorry!

File: src/config/quicksand.php (hunk 3 above)
Comment (LKDevelopment) [reply src/config/quicksand.php]: Sure

**Comment by mattstauffer** [comment]
@LKDevelopment This looks great!

One problem: This *adds* the logs to a custom log file, but it doesn't keep those logs from being written to the default laravel.log file.

Is that something you think you could tackle as a part of it? Thanks!

**Comment by LKDevelopment** [comment]
@mattstauffer Should be fixed now.  I get all Handlers from the Monolog instance, clear the handlers array, set a new Handler with Log::useFiles and reset all after the log is written :)

**Comment by mattstauffer** [comment]
@LKDevelopment Great! Nice work!

**Pull request merged**